py/desiutil/test/test_bitmask.py

```diff
@@ -6,8 +6,9 @@
                         print_function, unicode_literals)
 # The line above will help with 2to3 support.
 import unittest
-from ..bitmask import BitMask, _MaskBit
+from desiutil.bitmask import BitMask, _MaskBit
```

Comment:
You will now be able to do, *e.g.*, `python setup.py test -m desiutil.test.test_bitmask` in order to test individual modules.  There is simple, generic boilerplate that can be added to any test module to enable this.